device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/edge/DirectMethodResponse.java

```diff
@@ -43,16 +43,30 @@ public int getStatus()
         return this.status;
     }
 
+    /**
+     * Return the DirectMethodResponse payload in JsonElement type
+     * @return the DirectMethodResponse payload in JsonElement type
+     */
     public JsonElement getPayloadAsJsonElement()
     {
         return payload;
     }
 
-    public String getPayloadAsJsonString()
+    /**
+     * Return the DirectMethodResponse payload in String type
```

Comment:
Use this if you wish to deserialize to a specific type using a deserialization library of your choice.